src/pages/SimpleViewer.jsx

```diff
@@ -28,6 +29,48 @@ export default function SimpleViewer(props) {
   const [showSnackbar, toggleSnackbar] = useState(false);
   const messageStore = useMessageStore();
 
+  async function askQuestion() {
+    const defaultQuestion = {
+      parent: '',
+      visibility: 1,
```

Comment:
I think we should avoid using the visibility numbers. In utils/cache.js there is a mapping of numbers to strings for visibility. We should probably generate an inverse of that mapping and use that instead of the constant numbers.